code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -1071,3 +1071,23 @@
 		connected_ai.connected_robots |= src
 		notify_ai(ROBOT_NOTIFICATION_NEW_UNIT)
 		sync()
+
+//For picking up small animals
+/mob/living/silicon/robot/MouseDrop(atom/over_object)

```

Comment:
Was this tested?

Also, this doesn't look like it's for picking up small animals. And are you sure that this can't be tied to the recharge station? Because I /think/ it can be. If so, should be.